docs/modules/ROOT/pages/hazelcast-platform-operator-tls.adoc

```diff
@@ -0,0 +1,148 @@
+= Configure TLS with Hazelcast Platform Operator tutorial
+:page-layout: tutorial
+:page-product: operator
+:page-categories: Cloud Native
+:page-lang: go, java
+:page-enterprise: true
+:page-est-time: 10 mins
+:description: Learn how to secure Hazelcast cluster with TLS and Hazelcast Kubernetes Operator.
+
+{description}
+
+== Context
+In this tutorial, you'll do the following:
+
+- Create Kubernetes Secret with TLS certificate and key.
+
+- Deploy Hazelcast cluster with TLS (mTLS) encryption enforced between members.
+
+- Deploy Management Center to monitor your Hazelcast cluster.
+
+- Connect to you cluster using TLS to encrypt member to client network traffic.
+
+
+== Before you Begin
+
+Before starting this tutorial, make sure that you meet the following prerequisites:
+
+* Up and running https://kubernetes.io/[Kubernetes] cluster
+* Kubernetes command-line tool, https://kubernetes.io/docs/tasks/tools/#kubectl[kubectl]
+* Deployed xref:operator:ROOT:index.adoc[Hazelcast Platform Operator]
+* OpenSSL command-line tool, https://www.openssl.org/[openssl]
+
+== Step 1. Create Kubernetes Secret with TLS certificate and key
+
+. Generate TLS certyfcate and key in PEM format using OpenSSL
++
+[source, shell]
+----
+openssl req -x509 -sha256 -nodes -days 3650 -subj '/CN=example' -addext "subjectAltName=DNS:example" -newkey rsa:2048 -outform pem -keyout example.key -out example.crt -addext "extendedKeyUsage=serverAuth,clientAuth"
+----
+
+. Create Kubernetes Secret with TLS certificate and key
++
+[source, shell]
+----
+kubectl create secret tls example --cert=example.crt --key=example.key
+----
++
+NOTE: In production setup it is advised to use tools like https://cert-manager.io/[cert-manager] to manage your certificates.
+
+== Step 2. Deploy Hazelcast cluster with TLS
+
+. Create a secret with your link:http://trialrequest.hazelcast.com/[Hazelcast Enterprise License].
++
+[source, shell]
+----
+kubectl create secret generic hazelcast-license-key --from-literal=license-key=<hz-license-key>
+----
+
+. Create the Hazelcast cluster with TLS configured.
++
+[source, yaml]
+----
+kubectl apply -f - <<EOF
+include::ROOT:example$/hazelcast.yaml[]
```

Comment:
In the example hazelcast yaml we are using `unisocket` but in the sample client we are not specifying unisocket or disabling `smartRouting `. So either we should specify `exposeExternally` type as `Smart` and keep the sample clients as they are or we should keep the hazelcast yaml as it is and specify unisocket or disable `smartRouting` for the client configs.

You can check the proper configurations for hazelcast yaml and clients in our exposeExternally tutorial.
https://docs.hazelcast.com/tutorials/hazelcast-platform-operator-expose-externally